core/modules/inputmanager.js

```diff
@@ -0,0 +1,72 @@
+/*\
+title: $:/core/modules/inputmanager.js
+type: application/javascript
+module-type: global
+
+Input handling utilities
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+function InputManager(options) {
+	options = options || "";
+	this.inputs = [];
+}
+
+InputManager.prototype.findInputById = function(identifier) {
+	var result;
+	for(var i=0; i<this.inputs.length; i++) {
+		var object = this.inputs[i];
```

Comment:
why is it named `object` ... is there a better name? .... Is it an html `element` ?

Reply:
no it's just an object containing 
```
{
id: uniqueID,
selectionStart: value,
selectionEnd: value,
shoudFocusAgain: boolean
}
```

I think inputInfo is a good name